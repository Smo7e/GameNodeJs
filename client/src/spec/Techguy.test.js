import React from "react";
import { render } from "@testing-library/react";
import Techguy from "../component/Heroes/component/Techguy"; 

describe("Techguy Component", () => {
  it("renders correctly", () => {
    const Heroes = jest.fn(); 
    const { asFragment } = render(<Techguy heroes={Heroes} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
