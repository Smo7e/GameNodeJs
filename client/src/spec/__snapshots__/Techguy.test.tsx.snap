// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techguy Component renders correctly 1`] = `
<DocumentFragment>
  <div>
    <button
      class="arrow-3-heroes"
    />
    <div
      class="image-techguy-heroes"
    />
    <div
      class="panel-heroes"
    >
      <div
        class="text-heroes"
      >
        “Технарь”- имеет обычное здоровье в 100 единиц, увеличенный вдвое урон по парам “Математика” и “Программирование”. Предметы типа “Гаджет” улучшают характеристики в 1.5 раза сильнее.
      </div>
    </div>
  </div>
</DocumentFragment>
`;
